### ISSUE TITLE
Can't download TikTok videos

### ISSUE THREAD

**Issue by calielian** [JunkFood02/Seal#1699]
### Checklist

- [ ] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

I can't download videos from TikTok. 

### Error reports

App version: 1.13.0 (11302)
Device information: Android 13 (API 33)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.08.06

URL: https://vm.tiktok.com/ZMhLcX4oH/
[debug] Command-line config: ['--no-mtime', '--restrict-filenames', '-v', '--download-archive', '/data/user/0/com.junkfood.seal/files/archive.txt', '--no-playlist', '--downloader', 'libaria2c.so', '--external-downloader-args', 'aria2c:"--summary-interval=1"', '--add-metadata', '--no-embed-info-json', '-S', 'vcodec:vp9.2', '--embed-chapters', '-P', '/storage/emulated/0/Download/Seal', '-P', 'temp:/storage/emulated/0/Download/Seal/tmp', '-o', '%(title).200B.%(ext)s', '--no-cache-dir', '--ffmpeg-location', '/data/app/~~Ov2j9qIhiirYRqeI9sgTtA==/com.junkfood.seal-rLELR3ezQkDs0V9lEL-BaQ==/lib/arm64/libffmpeg.so', 'https://vm.tiktok.com/ZMhLcX4oH/']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8 (No ANSI), error utf-8 (No ANSI), screen utf-8 (No ANSI)
[debug] yt-dlp version stable@2024.08.06 from yt-dlp/yt-dlp [4d9231208] (zip)
[debug] Python 3.8.0 (CPython aarch64 64bit) - Linux-5.4.233-qgki-27101688-abM526BXXS5CXE2-aarch64-with-libc (OpenSSL 1.1.1t  7 Feb 2023, libc)
[debug] exe versions: ffmpeg 7.0.1 (setts), ffprobe 7.0.1
[debug] Optional libraries: Cryptodome-3.17, mutagen-1.46.0, sqlite3-3.30.1
[debug] Proxy map: {}
[debug] Request Handlers: urllib
[debug] Loaded 1830 extractors
[debug] Loading archive file '/data/user/0/com.junkfood.seal/files/archive.txt'
[debug] [TikTok] Found universal data for rehydration
[debug] Extractor gave empty title. Creating a generic title
[debug] Sort order given by user: vcodec:vp9.2
[debug] Formats sorted by: hasvid, ie_pref, vcodec:vp9.2(10), lang, quality, res, fps, hdr:12(7), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[debug] Invoking aria2c downloader on "https://www.tiktok.com/aweme/v1/play/?faid=1988&file_id=4ea898bc6d2b49c8a15338db09c2bc23&is_play_url=1&item_id=7399695666009640225&line=0&ply_type=2&signaturev3=dmlkZW9faWQ7ZmlsZV9pZDtpdGVtX2lkLmRiNDFkNDU4NmVjZmUxZTMyY2FmMGM4MDRhMDgzMjQx&tk=tt_chain_token&video_id=v0f044gc0000cqofvqfog65h7mcst91g"
[debug] ffmpeg command line: '/data/app/~~Ov2j9qIhiirYRqeI9sgTtA==/com.junkfood.seal-rLELR3ezQkDs0V9lEL-BaQ==/lib/arm64/libffmpeg.so' -y -loglevel repeat+info -i file:/storage/emulated/0/Download/Seal/tmp/TikTok_video_7399695666009640225.mp4 -map 0 -dn -ignore_unknown -c copy -write_id3v1 1 -metadata 'title=Bad bunny bebe remix' -metadata date=20240805 -metadata 'purl=https://www.tiktok.com/@jetnyan09/video/7399695666009640225?_t=8ppSfHuLIJh&_r=1' -metadata 'comment=https://www.tiktok.com/@jetnyan09/video/7399695666009640225?_t=8ppSfHuLIJh&_r=1' -metadata artist=ZafiroXs -movflags +faststart file:/storage/emulated/0/Download/Seal/tmp/TikTok_video_7399695666009640225.temp.mp4
[debug] ffmpeg version 7.0.1 Copyright (c) 2000-2024 the FFmpeg developers
  built with Android (9352603, based on r450784d1) clang version 14.0.7 (https://android.googlesource.com/toolchain/llvm-project 4c603efb0cca074e9238af8b4106c30add4418f6)
  configuration: --prefix=/home/runner/work/build-ffmpeg/build-ffmpeg/build/ffmpeg/arm64-v8a --enable-cross-compile --target-os=android --arch=aarch64 --sysroot=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/sysroot --cc=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android21-clang --cxx=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android21-clang++ --ld=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android21-clang --ar=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar --as=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android21-clang --nm=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-nm --ranlib=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ranlib --strip=/usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip --extra-cflags='-O3 -fPIC -I/home/runner/work/build-ffmpeg/build-ffmpeg/build/external/arm64-v8a/include' --extra-ldflags='-L/home/runner/work/build-ffmpeg/build-ffmpeg/build/external/arm64-v8a/lib -lm -lm -lm -lm -lsharpyuv -lz -lm -lz -lm -lc++' --enable-shared --disable-static --disable-vulkan --pkg-config=/usr/bin/pkg-config --enable-protocol=https --enable-version3 --enable-gpl --enable-libaom --enable-libdav1d --enable-libmp3lame --enable-libopus --enable-libtwolame --enable-libspeex --enable-libvpx --enable-libwebp --enable-libfreetype --enable-libfribidi --enable-mbedtls --enable-libbluray --enable-libxml2 --enable-libx264 --enable-libx265
  libavutil      59.  8.100 / 59.  8.100
  libavcodec     61.  3.100 / 61.  3.100
  libavformat    61.  1.100 / 61.  1.100
  libavdevice    61.  1.100 / 61.  1.100
  libavfilter    10.  1.100 / 10.  1.100
  libswscale      8.  1.100 /  8.  1.100
  libswresample   5.  1.100 /  5.  1.100
  libpostproc    58.  1.100 / 58.  1.100
[mov,mp4,m4a,3gp,3g2,mj2 @ 0xb400006f4fa9e3b0] Format mov,mp4,m4a,3gp,3g2,mj2 detected only with low score of 1, misdetection possible!
[mov,mp4,m4a,3gp,3g2,mj2 @ 0xb400006f4fa9e3b0] moov atom not found
[in#0 @ 0xb400006f0fa94bf0] Error opening input: Invalid data found when processing input
Error opening input file file:/storage/emulated/0/Download/Seal/tmp/TikTok_video_7399695666009640225.mp4.
Error opening input files: Invalid data found when processing input

ERROR: Postprocessing: Error opening input files: Invalid data found when processing input
Traceback (most recent call last):
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/YoutubeDL.py", line 3561, in process_info
    replace_info_dict(self.post_process(dl_filename, info_dict, files_to_move))
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/YoutubeDL.py", line 3743, in post_process
    info = self.run_all_pps('post_process', info, additional_pps=info.get('__postprocessors'))
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/YoutubeDL.py", line 3725, in run_all_pps
    info = self.run_pp(pp, info)
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/YoutubeDL.py", line 3703, in run_pp
    files_to_delete, infodict = pp.run(infodict)
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/postprocessor/common.py", line 23, in run
    ret = func(self, info, *args, **kwargs)
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/postprocessor/common.py", line 128, in wrapper
    return func(self, info)
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/postprocessor/ffmpeg.py", line 712, in run
    self.run_ffmpeg_multiple_files(
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/postprocessor/ffmpeg.py", line 330, in run_ffmpeg_multiple_files
    return self.real_run_ffmpeg(
  File "/data/user/0/com.junkfood.seal/no_backup/youtubedl-android/yt-dlp/yt-dlp/yt_dlp/postprocessor/ffmpeg.py", line 368, in real_run_ffmpeg
    raise FFmpegPostProcessorError(stderr.strip().splitlines()[-1])
yt_dlp.postprocessor.ffmpeg.FFmpegPostProcessorError: Error opening input files: Invalid data found when processing input

**Comment by Aijenbraotler** [comment]
Same bug to me :(

**Comment by Psychf** [comment]
Same issue

**Comment by Kuchinto** [comment]
Same problem here

**Comment by JunkFood02** [comment]
- https://github.com/yt-dlp/yt-dlp/issues/11034

TL;DR: Your IP is blocked by TikTok, and the app downloads the wrong files(instead of the video. Wait for a yt-dlp update or try changing your IP

**Comment by Ammar506tamim** [comment]
> * [[TikTok] 31KB non-video files are being downloaded yt-dlp/yt-dlp#11034](https://github.com/yt-dlp/yt-dlp/issues/11034)
> 
> TL;DR: Your IP is blocked by TikTok, and the app downloads the wrong files(instead of the video)
> 
> You can:
> 
> 1. Wait for a yt-dlp update
> 2. Try changing your IP
> 3. Try another format in the format selection page
The solution was tried and the same problem I tried to change IP on VPN.

**Comment by BreadJS** [comment]
Ytdl has updated it. Now it is seals turn :)
